app/messages/movie-quote

```diff
@@ -0,0 +1,2 @@
+By: <The Shawshank Redemption (1994)>
```

Comment:
Thanks, Refat! Can you please update your file so that the `By:` line contains your own name?

> By: Refat Hossen

Reply:
@SavvasStephanides  please check again. now update done